cdap-watchdog/src/test/java/co/cask/cdap/logging/framework/CDAPLogAppenderTest.java

```diff
@@ -213,6 +223,14 @@ public void testCDAPLogAppenderRotation() throws Exception {
       Assert.assertEquals(files.get(1).getEventTimeMs(), currentTimeMillisEvent1 + 1000);
       Assert.assertTrue(files.get(0).getFileCreationTimeMs() >= currentTimeMillisEvent1);
       Assert.assertTrue(files.get(1).getFileCreationTimeMs() >= currentTimeMillisEvent2);
+
+      // checking permission
+      String expectedPermissions = "rwxr-----";
```

Comment:
why rotation uses `rwxr-----`? Because `LogFileManager` will always create files with `600` permission right? We do not change rotated file permissions in `LogFileManager`